docs/introduction/Ecosystem.md

```diff
@@ -376,17 +359,6 @@ const newState = dotProp.set(state, `todos.${index}.complete`, true)
 const endOfArray = dotProp.get(obj, 'foo.$end')
 ```
 
-#### Immutable/Redux Interop
-
-**[gajus/redux-immutable](https://github.com/gajus/redux-immutable)** <br />
-combineReducers equivalent that works with Immutable.js Maps
-
-```js
-const initialState = Immutable.Map()
-const rootReducer = combineReducers({})
-const store = createStore(rootReducer, initialState)
-```
-
 **[eadmundo/redux-seamless-immutable](https://github.com/eadmundo/redux-seamless-immutable)** <br />
```

Comment:
:hand: We might as well remove this `seamless-immutable`-related lib too, since Immer obsoletes that also.

Reply:
There is also "rtfeldman/seamless-immutable" under "Data Structures", should we remove that too?

Reply:
Removed everything under the "Immutable Data" header, except Immer (Changed the link to Immer as well).